tensorboard/plugins/profile/profile_plugin.py

```diff
@@ -30,7 +30,6 @@
 from tensorflow.python.eager import profiler_client
 from tensorboard.backend import http_util
 from tensorboard.backend.event_processing import plugin_asset_util
-from tensorboard.plugins import base_plugin
```

Comment:
This dep is still needed for the definition of ProfilePlugin (which is the Travis failure)